airbyte-ci/connectors/ci_credentials/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
```

Comment:
I'm surprised to see this change in this PR. How does this relate to the .gitignore?

Reply:
It's because all `__init__.py` were originally ignored from formatting. This PR changes these rules: `__init__.py` files will be formatted if they're not gitignored. I removed these formatting change from this PR and merged the following ones:
* https://github.com/airbytehq/airbyte/pull/33286 
* https://github.com/airbytehq/airbyte/pull/33250

Reply:
I reverted these change. 
Our `__init__.py` files in the repo are not always properly formatted.
We have a clash between `black` formatter and `flake` linter for CDK files.
I opened a follow up issue to address this problem and continued to ignore `__init__.py` in this PR.